esx_service/cli/vmdkops_admin.py

```diff
@@ -236,9 +236,8 @@ def commands():
                     'func': tenant_rm,
                     'help': 'Delete a tenant',
                     'args': {
-                        '--name': {
+                        'name': {
```

Comment:
@lipingxue We all agree that local consistency is more important. For AdminCLI commands, the only inconsistent command is "policy rm". So we should just fix that command to use "--name" instead of fixing all other commands to use positional name.